### PR TITLE
[DOCS-447] Updated Device Handler Doc with Virtual Devices

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Modified the Device Handler Quick Start section with testing by a Virtual Device.

@jimmyjames the 2nd Acceptance Criterion of the DOCS-447 isn't really addressed. Love to hear your comments.

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
@jimmyjames Revised the content. Please check.

File: device-type-developers-guide/quick-start.rst [hunk 1]
```diff
@@ -15,136 +18,126 @@ This section will walk you through getting your first device handler running.
...
-Towards the bottom of the page for the CentraLite Switch, click the "Edit" button.
-Find the "Type" field, and select your Device Handler. Click the "Update" button to save your changes.
+----
 
-Your switch is now using your Device Handler.
-If you refresh the mobile app (you may need to kill it and restart it), you should see the tile icons updated with whatever label you gave it.
+.. note::
 
-If you make future changes to your device handler, don't forget to click the "Publish" button after you have saved.
+  While the simulator is still the preferred option for live logging and debugging your Device Handler or SmartApp, Virtual Devices are a quicker way to test the tiles of your Device Handler.
```
Comment (jimmyjames) [inline device-type-developers-guide/quick-start.rst]: Maybe not "preferred"... How about: "While the simulator is useful and necessary for testing how the Device Handler handles incoming messages, it is recommended to test on the mobile app whenever possible."

**Comment by rajkaramchedu** [comment]
@jimmyjames Done.

**Comment by jimmyjames** [comment]
👍

**Pull request merged**